accessible_api/AccessibilityAPIServer.py

```diff
@@ -0,0 +1,42 @@
+""" API for Accessible Requests """
+
+from SocketServer import TCPServer
+from argparse import ArgumentParser
+from AccessibilityRequestHandler import PlatformAccessibilityRequestHandler
```

Comment:
nit: here and in many other files lets use absolute imports instead of relative.